game.js

```diff
@@ -0,0 +1,82 @@
+let playerPoints = 0;
+let computerPoints = 0;
+const choices = ["Rock", "Paper", "Scissors"];
+
+// Here we taking the choice of computer
+const getComputerChoice = () => choices[Math.floor(Math.random()*choices.length)];
+
+// Here we are Playing a round of game between Player and Computer
+playRound = (playerSelection, computerSelection) => {
+    const playerChoice = playerSelection.toLowerCase();
+    const computerChoice = computerSelection.toLowerCase();
+    switch(playerChoice){
+        case 'rock':
+            switch(computerChoice){
+                case 'paper':
+                    return ("Computer");
+                case 'scissors':
+                    return ("Player");
+                default:
+                    return ("Tie");
+            }
+        case 'paper':
+            switch(computerChoice){
+                case 'scissors':
+                    return ("Computer");
+                case 'rock':
+                    return ("Player");
+                default:
+                    return ("Tie");
+            }
+        case 'scissors':
+            switch(computerChoice){
+                case 'rock':
+                    return ("Computer");
+                case 'paper':
+                    return ("Player");
+                default:
+                    return ("Tie");
+            }
+        default:
+            return ("Invalid Choice");
```

Comment:
In these return statements, adding a bracket is not necessary. 
You can directly return the string - return ".........."